sass/sass_wrapper.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * A Sass compiler wrapper that supports bazel persistent worker protocol.
+ * 
+ * Bazel can spawn a persistent worker process that handles multiple invocations. It
+ * will also be backwards compatible with being run as a normal single-invocation command.
+ */
+const {debug, log, runAsWorker, runWorkerLoop} = require('./worker');
+const sass = require('sass');
+const fs = require('fs');
+
+// Equivalent of running node with --expose-gc
+// but easier to write tooling since we don't need to inject that arg to
+// nodejs_binary
+if (typeof global.gc !== 'function') {
+  require('v8').setFlagsFromString('--expose_gc');
+  global.gc = require('vm').runInNewContext('gc');
+}
+
+function main(args) {
+  if (runAsWorker(args)) {
+    debug('Starting Sass compiler persistent worker...');
+    runWorkerLoop(runOneBuild);
+    // Note: intentionally don't process.exit() here, because runWorkerLoop
+    // is waiting for async callbacks from node.
+  } else {
+    debug('Running a single build...');
+    if (args.length === 0) throw new Error('Not enough arguments');
+    if (args.length !== 1) {
+      console.error('Expected one argument: path to flagfile');
+      return 0;
+    }
+    // Bazel worker protocol expects the only arg to be @<path_to_flagfile>.
+    // When we running a single build, we remove the @ prefix and read the list of
+    // actual arguments line by line.
+    const configFile = args[0].replace(/^@+/, '');
+    const configContent = fs.readFileSync(configFile, 'ascii').trim();
+    if (!runOneBuild(configContent.split('\n'))) {
```

Comment:
Why do this check if `runOneBuild` always returns `true`? Why not just call `sass.run_()` directly?

Reply:
Done.

Reply:
Still needs to be done.

Reply:
Done